web/profiles/joinup/config/install/views.view.collections.yml

```diff
@@ -0,0 +1,452 @@
+langcode: en
```

Comment:
I'm confused about this. Did this appear in the export? This is a view showing collections. I think this belongs to a different ticket, probably ISAICP-2205. It doesn't belong in the Joinup install profile either, this should be in the Collections module.

We have to be careful. If we delete it here then it's possible the view will disappear when this is merged with ISAICP-2205.

Can you investigate what happened?